react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -406,6 +406,7 @@ const axes = {
 const north_arrow_layer = {
     "@@type": "NorthArrow3DLayer",
     id: "north-arrow-layer",
+    color: [0, 0, 0, 255],
```

Comment:
Please also update the ["Axes / Dark Mode" story](https://equinor.github.io/webviz-subsurface-components/storybook-static/?path=/story/deckglmap-axes--dark-mode)